charts/incubator/blocky/questions.yaml

```diff
@@ -120,6 +120,35 @@ questions:
                                   schema:
                                     type: int
                                     default: 53
+        - variable: k8sgateway
+          label: k8s-Gateway DNS UPD Service
+          description: The k8s-Gateway DNS UDP service
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+# Include{serviceSelectorSimple}
+# Include{serviceSelectorExtras}
```

Comment:
can we keep this one clusterIP by default?
It's not used in most usecases anyway